numpy/doc/constants.py

```diff
@@ -175,109 +175,6 @@ def add_newdoc(module, name, doc):
 
     """)
 
```

Comment:
I think `numpy/doc` can be removed completely as well - anything that looks useful still and isn't already present can be moved to the html docs (it can't be much), and then it's `rm -rf` time I think. It shouldn't be used in any upstream library I think.

Reply:
In the `numpy/doc`, only two files were left: `constants.py` and `ufuncs.py`. I adjusted and moved the first one to the `constants.rst` file. I left the second one as, according to https://github.com/numpy/numpy/pull/17222, it still needs to be properly merged with `doc/source/reference/ufuncs.rst`.